### PR TITLE
ETS importer fix

### PR DESCRIPTION
**Pull request by MineTech0**

### CODE REVIEW

**Comment by ghost** [comment]
Please, only import models from mb.models subpackage. Otherwise code will break as it has done before.

File: app/tests/imports/test_pa_import.py
Comment (ghost) [inline app/tests/imports/test_pa_import.py:1]: Imports should be fixed on this. Check pylint for correct order and delete 
`from mb.models.location_models import SourceLocation` alltogether. Its already imported correctly.

Comment (ghost) [reply app/tests/imports/test_pa_import.py:1]: Also:
`import imports.views as views` this is no go. Import only what you need.
Double import:
`import pandas as pd`

**Review by ghost — APPROVED** [review]
OK. Pylint gives quite many errors, but those can be dealt with later on.

File: app/mb/templates/import/import_ets.html
Comment (A00N) [inline app/mb/templates/import/import_ets.html:1]: Joku tietty syy miks ETS onki Occurrences Data?

**Review by A00N — APPROVED** [review]

**Review by masiro918 — COMMENTED** [review]
Miksi mulle tulee virheilmoitus: ModuleNotFoundError: No module named 'fuzzywuzzy'. Puuttuukohan multa joku? Vaihdoin siis tähän brancheen ja yritin testata ETS importteria

**Comment by ghost** [comment]
> Miksi mulle tulee virheilmoitus: ModuleNotFoundError: No module named 'fuzzywuzzy'. Puuttuukohan multa joku? Vaihdoin siis tähän brancheen ja yritin testata ETS importteria

Try running `docker compose build`. There is updated `requiremets.txt` in main.

**Pull request merged**